.changeset/chilled-beers-run.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rocket.chat/i18n': patch
+'@rocket.chat/meteor': patch
```

Comment:
```suggestion
'@rocket.chat/i18n': minor
'@rocket.chat/meteor': minor
```
Since this is not a fix, these should be `minor` changes. Shouldn't they?